share/completions/brightnessctl.fish

```diff
@@ -0,0 +1,3 @@
+complete -f brightnessctl
+complete -k -c brightnessctl -n "test (__fish_number_of_cmd_args_wo_opts) = 1" \
```

Comment:
Please remove the `-k` (short for `--keep-order`) unless you have a good reason to keep it.

The default ordering should be simple, predictable and therefore alphabetical.

Reply:
Done